Tests/MeiliSearchIntegrationTests/SearchTests.swift

```diff
@@ -451,7 +451,7 @@ class SearchTests: XCTestCase {
     typealias MeiliResult = Result<SearchResult<Book>, Swift.Error>
```

Comment:
The description above
```
    let expectation = XCTestExpectation(description: "Search for Books using matches")
```

should become using showMatchesPosition. Sorry I can't make a suggestion on it :(